coalib/bears/PackageRequirement.py

```diff
@@ -0,0 +1,22 @@
+from coala_decorators.decorators import generate_repr
+
+
+@generate_repr()
+class PackageRequirement:
+
+    def __init__(self, manager: str, package: str, version=""):
+        """
+        :param manager: A string with the name of the manager (pip, npm, etc).
+        :param package: A string with the name of the package to be installed.
+        :param version: A number that contains the version. Leave empty to
+                        install the latest.
+        """
+        self.manager = manager
+        self.package = package
+        self.version = version
+
+    def check(self):
+        """
+        Check if the requirement is satisfied.
+        """
```

Comment:
Oh I think you should add a `:return:` spec, so definitely stating "returns `True` if satisfied, `False` if not", as this function gets overriden it :+1: